config/eigendaflags/v2/cli.go

```diff
@@ -0,0 +1,238 @@
+			EnvVars:  []string{withEnvPrefix(envPrefix, "DISPERSE_BLOB_TIMEOUT")},
+			Category: category,
+			Required: false,
+			Value:    time.Minute * 2,
+		},
+		&cli.DurationFlag{
+			Name:     BlobCertifiedTimeoutFlagName,
+			Usage:    "Maximum amount of time to wait for blob certification against the on-chain CertVerifier.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CERTIFY_BLOB_TIMEOUT")},
+			Category: category,
+			Required: false,
+			Value:    time.Second * 30,
+		},
+		&cli.BoolFlag{
+			Name:     V2EnabledFlagName,
+			Usage:    "Enable blob dispersal and retrieval against EigenDA v2 protocol.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "V2_ENABLED")},
+			Category: category,
+			Required: false,
+		},
+		&cli.StringFlag{
+			Name:     CertVerifierAddrFlagName,
+			Usage:    "Address of the EigenDACertVerifier contract. Required for performing eth_calls to verify EigenDA certificates.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CERT_VERIFIER_ADDR")},
+			Category: category,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     ContractCallTimeoutFlagName,
+			Usage:    "Timeout used when performing smart contract eth_calls.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CONTRACT_CALL_TIMEOUT")},
+			Category: category,
+			Value:    10 * time.Second,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     RelayTimeoutFlagName,
+			Usage:    "Timeout used when querying a relay for blob contents.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "RELAY_TIMEOUT")},
+			Category: category,
+			Value:    10 * time.Second,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     BlobStatusPollInterval,
+			Usage:    "Duration to query for blob status updates during dispersal.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "BLOB_STATUS_POLL_INTERVAL")},
+			Category: category,
+			Value:    1 * time.Second,
+			Required: false,
+		},
+		&cli.UintFlag{
+			Name:     BlobVersionFlagName,
+			Usage:    "Blob version used when dispersing. Currently only supports (0).",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "BLOB_VERSION")},
+			Category: category,
+			Value:    uint(0),
+			Required: false,
+		},
+	}
+}
+
+func ReadConfig(ctx *cli.Context) common.V2ClientConfig {
+	return common.V2ClientConfig{
+		Enabled:               ctx.Bool(V2EnabledFlagName),
+		ServiceManagerAddress: ctx.String(SvcManagerAddrFlagName),
+
+		DisperserClientCfg: readDisperserCfg(ctx),
+		PayloadClientCfg:   readPayloadDisperserCfg(ctx),
+		RetrievalConfig:    readRetrievalConfig(ctx),
+		PutRetries:         ctx.Uint(PutRetriesFlagName),
+	}
+}
+
+func readPayloadClientConfig(ctx *cli.Context) v2_clients.PayloadClientConfig {
+	noPolynomial := ctx.Bool(DisablePointVerificationModeFlagName)
+	polyMode := codecs.PolynomialFormCoeff
+
+	// if point verification mode is disabled then blob is treated as evaluations and
+	// not FFT'd before dispersal
+	if noPolynomial {
+		polyMode = codecs.PolynomialFormEval
+	}
```

Comment:
`polyMode` is not very precise here. Let's stick to `PayloadPolynomialForm`

Reply:
[20944e3](https://github.com/Layr-Labs/eigenda-proxy/pull/296/commits/20944e32884209950be9ff3c4ef017945ca3fc94)